packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -84,13 +86,27 @@ export class SOQLEditorInstance {
     webviewPanel.onDidDispose(this.dispose, this, this.subscriptions);
   }
 
+  /* --- MESSAGING TO THE UI ---- */
+
   protected updateWebview(document: vscode.TextDocument): void {
     this.webviewPanel.webview.postMessage({
       type: MessageType.TEXT_SOQL_CHANGED,
       payload: document.getText()
     });
   }
 
+  protected updateDocumentInfo(document: vscode.TextDocument): void {
+    const name = getDocumentName(document);
+    const payload = {
+      name
+    };
+
+    this.webviewPanel.webview.postMessage({
+      type: MessageType.DOCUMENT_INFO,
+      payload
+    });
+  }
+
```

Comment:
This method can also be used in the future as a miscellaneous bucket to send any other information we need to the UI about the text document.